README.markdown

```diff
@@ -75,6 +75,13 @@ The `stdlib::stages` class declares various run stages for deploying infrastruct
 
 ### Types
 
+#### `absolutepath`
+
+A strict absolute path type. Uses a Variant of Unixpath and Windowspath types.
+
+Acceptable input examples:    /usr2/username/bin:/usr/local/bin:/usr/bin:.
```

Comment:
This example is not listed in the tests, but should be, to avoid breaking the README ;-)